lib/k8s/client/mint_http_provider.ex

```diff
@@ -194,7 +194,7 @@ defmodule K8s.Client.MintHTTPProvider do
   @spec get_content_type(keyword()) :: binary | nil
   defp get_content_type(headers) do
     case List.keyfind(headers, "content-type", 0) do
-      {_key, content_type} -> content_type
+      {_key, content_type} -> content_type |> String.split(";") |> List.first()
```

Comment:
can you elaborate on this?

Reply:
The azure end point is returning a response with content type `application/json; charset=utf-8` since the charset is not needed for the parsing I just threw it away